src/js/components/DataTable/Cell.js

```diff
@@ -30,7 +30,7 @@ const Cell = ({
   const context = useContext(TableContext);
 
   let content;
-  if (render) {
+  if (render && datum[property]) {
```

Comment:
What if the caller wants their custom renderer to provide some indication that there is no data in this cell? I'm wondering if detecting a lack of the property should be the purview of the render function.

Reply:
Good point. I can see use cases where identifying missing data would be useful. In that case, need to revisit how the footer values are being assembled.